Common/src/CConfig.cpp

```diff
@@ -4314,6 +4364,702 @@ void CConfig::SetPostprocessing(unsigned short val_software, unsigned short val_
              ( Kind_Solver == INC_NAVIER_STOKES      ) ||
              ( Kind_Solver == INC_RANS               ) );
 
+  /*--- Reacting flows iniatilization ---*/
+  if (( Kind_Solver == NEMO_EULER             ) ||
+      ( Kind_Solver == NEMO_NAVIER_STOKES     ) )
+      /*( Kind_Solver == NEMO_RANS              )) */{
+
+    bool init_err;
+    unsigned short maxEl = 0;
+    unsigned short iSpecies, jSpecies, iEl;
+    su2double mf;
+
+    switch (Kind_GasModel) {
+    case ONESPECIES:
+      /*--- Define parameters of the gas model ---*/
```

Comment:
I would move this chunk of code to a separate method of config

Reply:
I think this will be temporary as we move these routines to the fluid_model routines.

Reply:
More reason to make it as independent as possible.

Reply:
@Pedro, this is one of the steps we currently doing by moving to FluidModel, we are all agreeing here :)